CHANGES.md

```diff
@@ -25,12 +26,19 @@ OpenSSL Releases
  - [OpenSSL 1.0.0](#openssl-100)
  - [OpenSSL 0.9.x](#openssl-09x)
 
-OpenSSL 3.5
+OpenSSL 3.6
 -----------
 
 ### Changes between 3.5 and 3.6 [xx XXX xxxx]
 
- * none yet
+* Support setting a free function thunk to OPENSSL_sk stack types. Using a thunk
+  allows the type specific free function to be called with the correct type
+  information from generic functions like OPENSSL_sk_pop_free().
+
+  *Frederik Wedel-Heinen*
```

Comment:
Please indent these lines by one more space. I am not quite sure this deserves a CHANGES.md entry buy yeah ok.

Reply:
If it's user visible, it should get a changes entry IMO.

Reply:
Speaking of the indentation, it seems like we already have some confusion in the 3.5 section (below).